Touch.Server/Main.cs

```diff
@@ -99,20 +99,23 @@ public int Start ()
 		return 0;
 	}
 
-	public bool Processing (TcpClient client)
+	public bool Processing (TcpClient client, bool skipHeader = false)
 	{
 		string logfile = Path.Combine (LogPath, LogFile ?? DateTime.UtcNow.Ticks.ToString () + ".log");
 		string remote = client.Client.RemoteEndPoint.ToString ();
 		Console.WriteLine ("Connection from {0} saving logs to {1}", remote, logfile);
 		connected.Set ();
 
 		using (FileStream fs = File.OpenWrite (logfile)) {
-			// a few extra bits of data only available from this side
-			string header = String.Format ("[Local Date/Time:\t{1}]{0}[Remote Address:\t{2}]{0}", 
-				Environment.NewLine, DateTime.Now, remote);
-			byte[] array = Encoding.UTF8.GetBytes (header);
-			fs.Write (array, 0, array.Length);
-			fs.Flush ();
+			if (!skipHeader)
+			{
+                // a few extra bits of data only available from this side
```

Comment:
The coding style is to add the opening brace at the end of the `if` line:

```csharp
if (...) {
```

Also you're reformatting the subsequent lines (replacing tabs with spaces, removing spaces before the opening parenthesis in method calls, etc.). Please undo those changes.